chaos.proto

```diff
@@ -69,6 +69,9 @@ message Impact{
     repeated PtObject informed_entities = 6;
 
     repeated Message messages = 7;
+
+    // Send a message
+    optional bool push = 8;
```

Comment:
two point: 
  - don't you want a datetime in place of a boolean? this way you can plan a push in advance
  - if you use a date, maybe it will be better to put this property in the message, so you won't have the obligation to push all messages at the same time